specs/lightclient/beacon-chain.md

```diff
@@ -284,13 +285,10 @@ def get_sync_committee(state: BeaconState, epoch: Epoch) -> SyncCommittee:
     Return the sync committee for a given state and epoch.
     """
     indices = get_sync_committee_indices(state, epoch)
-    validators = [state.validators[index] for index in indices]
-    pubkeys = [validator.pubkey for validator in validators]
-    aggregates = [
-        bls.AggregatePKs(pubkeys[i:i + SYNC_SUBCOMMITTEE_SIZE])
-        for i in range(0, len(pubkeys), SYNC_SUBCOMMITTEE_SIZE)
-    ]
-    return SyncCommittee(pubkeys=pubkeys, pubkey_aggregates=aggregates)
+    pubkeys = [state.validators[index].pubkey for index in indices]
```

Comment:
Note, this is a cleanup from #2212 that is being ported as well. Should be 1:1 functionality